### PR TITLE
Refactor CSS and menu code for improved responsiveness and readability

### PR DESCRIPTION
**Pull request by Nortalle**

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -65,13 +65,21 @@
 	color: #e53935;
 }
 
-@media (max-width: 991px) {
+@media (max-width: 1024px) {
```
Comment (Nortalle) [inline public/stylesheets/style.css:68]: Harmonization to picocss breakpoints

File: views/partials/menu.njk [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<ul
+    dir="rtl">
```
Comment (ludelafo) [inline views/partials/menu.njk:2]: Why is that here?
